Tools/LambdaTestTool-v2/src/Amazon.Lambda.TestTool/README.md

```diff
@@ -0,0 +1,298 @@
+# AWS Lambda Test Tool
+
+## Overview
+The AWS Lambda Test Tool provides local testing capabilities for .NET Lambda functions with support for both Lambda emulation and API Gateway emulation. This tool allows developers to test their Lambda functions locally in three different modes:
+
+1. Lambda Emulator Mode
+2. API Gateway Emulator Mode
+3. Combined Mode (both emulators)
+
+## Comparison with Previous Test Tool
+
+The AWS Lambda Test Tool is an evolution of the previous AWS .NET Mock Lambda Test Tool, with several key improvements:
+
+### New Features
```

Comment:
anything else worth mentioning here?

Reply:
* New flow for loading Lambda functions that mimic's closer to the Lambda service. This solves many of the issues with the older tool when it came to loading dependencies.
* Ability to have multiple Lambda functions use the same instance of the test tool.
* UI refresh
* Support for integration with .NET Aspire (https://github.com/aws/integrations-on-dotnet-aspire-for-aws/issues/17)

Reply:
updated